tests/snappi_tests/multidut/pfc/test_multidut_pfc_pause_lossless_with_snappi.py

```diff
@@ -257,6 +257,7 @@ def test_pfc_pause_single_lossless_prio_reboot(snappi_api,                  # no
 
     for duthost in [snappi_ports[0]['duthost'], snappi_ports[1]['duthost']]:
         logger.info("Issuing a {} reboot on the dut {}".format(reboot_type, duthost.hostname))
+        duthost.shell("sudo config save -y")
```

Comment:
when test finishes, we need to restore config_db back to it was. Otherwise, the config-db will have residual IP address after the test.
either save the config again in cleanup_config, or use some fixture to save the config before the test, and restore it after the test?

Reply:
@sdszhang , the cleanup is already being done by the teardown/golden-running-config recovery mechanism.

Reply:
Are you referring to core_dump_and_config_check? I think it checks for running-config only, not config-db file. and it takes around 10 minutes to recover on T2 chassis if there is running-config mismatch. If we do the config save during or after cleanup, it will be quicker.